frontend/src/components/pages/StudentAssessment.tsx

```diff
@@ -0,0 +1,217 @@
+/* eslint-disable react/prop-types */
+import React from "react";
+import { ChakraProvider, Box, Text, Stack, Button } from "@chakra-ui/react";
+import {
+  JUMP_MATH_LOGO,
+  MULTI_SELECT,
+  MULTI_CHOICE,
+  SHORT_ANSWER,
+} from "../../assets/images";
+import theme from "../../themes";
+
+type BoxProps = {
+  children: React.ReactNode;
+  top?: string;
+  left?: string;
+};
+
+const InnerBox: React.FC<BoxProps> = ({ children, top, left }) => (
+  <Box
+    backgroundColor="blue.50"
+    borderRadius="10px"
+    position="absolute"
+    padding="10px"
+    gap="40px"
+    width="115px"
+    height="115px"
+    top={top}
+    left={left}
+  >
+    {children}
+  </Box>
+);
+
+const OutterBox: React.FC<BoxProps> = ({ children, top, left }) => (
+  <Box
+    backgroundColor="rgba(232, 237, 241, 0.3)"
+    borderRadius="10px"
+    position="absolute"
+    padding="32px"
+    gap="40px"
+    width="480px"
+    height="376px"
+    top={top}
+    left={left}
+  >
+    {children}
+  </Box>
+);
+
+const assessmentMetadata = {
+  numOfQuestions: 12,
+  totalPoints: "50 + 1 (Bonus)",
+  questionTypes: ["Multiple choice", "Multi-Select", "Short Answer"],
+  rules:
+    "The test WILL be monitored so please close any windows before starting the test. \nYou will have 1 hour to complete this test. No aids are permitted. \nIf you need clarification or assistance, please raise your hand quietly and I will come to you. \nGood Luck! \n- Mr. Roberts",
+  questionText: "Number of Questions",
+  totalPointsText: "Total Points",
+  questionTypesText: "Question Types",
+  startTime: "Start Time: September 15, 2022 at 2:00pm",
+  test: "Unit 0 Review Test",
+};
+
+const questionTypeImages = assessmentMetadata.questionTypes.map(
+  (type, index) => {
+    switch (type) {
+      case "Multiple choice":
+        return (
+          <InnerBox key={index} top="190px" left={`${35 + index * 147}px`}>
+            <img
+              src={MULTI_CHOICE.src}
+              alt="multi-choice"
+              style={{ width: "65px", marginTop: "17px", marginLeft: "15px" }}
+            />
+          </InnerBox>
+        );
+      case "Multi-Select":
+        return (
+          <InnerBox key={index} top="190px" left={`${35 + index * 140}px`}>
+            <img
+              src={MULTI_SELECT.src}
+              alt="multi-select"
+              style={{ width: "60px", marginTop: "20px", marginLeft: "20px" }}
+            />
+          </InnerBox>
+        );
+      case "Short Answer":
+        return (
+          <InnerBox key={index} top="190px" left={`${35 + index * 144}px`}>
+            <img
+              src={SHORT_ANSWER.src}
+              alt="short-answer"
+              style={{ width: "60px", marginTop: "20px", marginLeft: "18px" }}
+            />
+          </InnerBox>
+        );
+      default:
+        return null;
+    }
+  },
+);
+
+const AssessmentSummary = (): React.ReactElement => {
+  return (
+    <ChakraProvider theme={theme}>
```

Comment:
```suggestion
    <>
```
you can use empty tags for that ! `<>...</>`